Raven.Tests.Issues/RavenDB_4836.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.Composition.Hosting;
+using System.Linq;
+using Lucene.Net.Analysis;
+using Lucene.Net.Analysis.BR;
+using Lucene.Net.Documents;
+using Raven.Abstractions.Data;
+using Raven.Abstractions.Indexing;
+using Raven.Client.Bundles.MoreLikeThis;
+using Raven.Client.Embedded;
+using Raven.Client.Indexes;
+using Raven.Database.Indexing;
+using Raven.Database.Plugins;
+using Raven.Tests.Helpers;
+using Xunit;
+
+namespace Raven.Tests.Issues
+{
+    public class RavenDB_4836 : RavenTestBase
+    {
+        private readonly EmbeddableDocumentStore _documentStore;
+
+        public RavenDB_4836()
+        {
+            _documentStore = NewDocumentStore();
```

Comment:
Yes, this is right. 
But it safer this way, and make it easy to port to future versions.